### PR TITLE
Added flytectl config by env variable 

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
- Fix documentation error of default config 
- override config by env variable 

## Type
 - [x] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature, etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1121
https://github.com/flyteorg/flyte/issues/1120

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Review by kumare3 — DISMISSED** [review]

**Review by EngHabu — DISMISSED** [review]
Thank you

**Comment by evalsocket** [comment]
@EngHabu @kumare3 You need to review again. My one commit was not verifed

**Review by kumare3 — APPROVED** [review]
